cmd/cloudshell_open/scripts.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/fatih/color"
+)
+
+type myWriter struct {
+	out io.Writer
+	color color.Attribute
+}
+
+func (m myWriter) Write(p []byte) (int, error) {
+	return color.New(m.color).Fprintf(m.out, string(p))
+}
+
+func runScript(dir, command string, envs []string) error {
+	fmt.Println(infoPrefix + " Running command: " + color.BlueString(command))
+
+	cmd := exec.Command("/bin/bash", "-c", "set -euo pipefail; set -x; " + command)
+	cmd.Env = envs
+	cmd.Dir = dir
+	cmd.Stdout = myWriter{os.Stdout, color.FgGreen}
+	cmd.Stderr = myWriter{os.Stderr, color.FgRed}
```

Comment:
I think stderr==red rarely works out for good. It scares users much more than it helps, even curl prints progress to stderr, and we don't want all that to be red. You can fix later.

It would be better if these were just gray (FgHiBlack)

Reply:
That makes sense.  Thanks!